.github/workflows/sonar-scan.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           uses: JJ/github-pr-contains-action@releases/v7
           with:
               github-token: ${{ secrets.GITHUB_TOKEN }}
-              bodyDoesNotContain: "[\"|`]"
+              bodyDoesNotContain: "[\"|\']"
```

Comment:
Let's unescape ` and just leave it for now. We might have to pull this action since it doesn't seem to work.